util/cron/test-linux64-llvm15.bash

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# Test default configuration on examples only, on linux64, with llvm 15
+
+export OFFICIAL_SYSTEM_LLVM=true
+source /data/cf/chapel/setup_system_llvm.bash 15
```

Comment:
@riftEmber do we have this script working with the new Spack build of LLVM 15? Or does this need to do something else?

Reply:
Not using spack. My understanding (which may have been incorrect) was that Bhavani and I were planning to work together on hooking up the Spack build of LLVM 15 to be used before these new configs went in. I didn't actually realize there even was a manual build of LLVM 15 here. In any case I can just change it to use the spack build after the fact.